lib/test-data.json

```diff
@@ -57,7 +71,9 @@
       "outputs": [
         {
           "id": "2-output-g",
-          "label": "My output"
+          "label": "My output",
```

Comment:
A small, and probably insignificant note - there is currently no requirement or plan to make labels distinct from ids.